cmd/serviceaccount-secret-rotation-trigger/main.go

```diff
@@ -64,7 +71,7 @@ func main() {
 	for clusterName, config := range kubeconfigs {
 		clusterName, config := clusterName, config
 		eg.Go(func() error {
-			client, err := ctrlruntimeclient.New(config, ctrlruntimeclient.Options{})
+			client, err := ctrlruntimeclient.New(&config, ctrlruntimeclient.Options{})
```

Comment:
This takes the address of the loop variable `config` which gets reset at every iteration. Pass it as a parameter to the anonymous function given to `eg.Go()`

Reply:
No, it gets shadowed in line 72

Reply:
ffs. I really dislike that pattern, it's super hard to see and read :eyeroll: